content/programming-cookbook/vs-code/remote.md

```diff
@@ -24,7 +24,7 @@ showToc: true
 
 # ก่อนไปลุยทำความเข้าใจมันนิดว่ามันทำงานยังไง
 
-![][architecture]
+![architecture]
```

Comment:
I haven't known if you use reference image, you can omit square bracket of alternative text.